com.gitlab.sat_metalab.Splash.json

```diff
@@ -95,21 +95,20 @@
         {
             "name": "splash",
             "buildsystem": "simple",
-            "config-opts": [
-                "-DBUILD_GENERIC_ARCH=ON"
-            ],
             "build-commands": [
-                "./make_deps.sh",
+                "git submodule update --init --recursive",
```

Comment:
this is not supposed to work or do anything. At this stage the submodule should be properly set, and there is no network anyway to fetch anything else.